src/Functions/ggmlEvaluate/model_storage.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
```

Comment:
Please use CamelCase for file names. Style guide: https://clickhouse.com/docs/en/development/style